### DISCUSSION TITLE
markdown vs org-mode

### DISCUSSION THREAD

**Discussion by gour** [ahyatt/ekg#138] [Q&A] [answered]
Hello,

I'm curious what would be pro/cons in using markdown-mode vs. org-mode for taking notes within ekg?

I understand that org-mode is much more extensive markup, but, maybe, for the short notes markdown can do the job.

Moreover, Andrew does explain in his videos that there is ongoing work to adjust ekg for handling org-mode buffers...

**Comment by ahyatt** [comment, marked as answer]
I think if you aren't planning to take advantage of org-mode features, markdown is pretty good.  In fact, if you plan on using the LLM capabilities, they just are much better at markdown formatting than org-mode formatting.

On the other hand, I don't know how much people are regularly using markdown, so you may discover fun new bugs!

**Reply by gour** [reply]
>  In fact, if you plan on using the LLM capabilities, they just are much better at markdown formatting than org-mode formatting.

Hmm...that's interesting! I haven't tried LLM yet, but it is good to know that. Thank you.

> On the other hand, I don't know how much people are regularly using markdown, so you may discover fun new bugs!

😊

**Comment by jayrajput** [comment]
`org-mode` is generally better than `markdown` in my personal view. The `org-mode` is not only markup, it is much more. For example: I have live code embedded in my `org-mode` ekg notes.
